### PR TITLE
New operator spacing rule for eslint

### PR DESCRIPTION
**Pull request by allen-tran**
### What does this do?
Previously [operators](https://www.w3schools.com/js/js_operators.asp) such as the `=` sign were either spaced evenly or not spaced at all. There was no consistency as files were commonly seen with this mismatched pattern.

See below for previous

2021-12-30 at 1 30 24 PM

See below for new

2021-12-30 at 1 32 06 PM

See below for error thrown with new config

2021-12-30 at 1 30 33 PM

### Features
- [X] includes new rule in eslint config file
- [X] enforces and corrects spacing rule across C4

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -56,11 +56,14 @@
         ],
         "no-extra-semi": "error",
         "eol-last": 2,
-        "space-before-function-paren": ["error", {
-          "anonymous": "never",
-          "named": "never",
-          "asyncArrow": "always"
-        }],
+        "space-before-function-paren": [
```
Comment (allen-tran) [inline .eslintrc.json:59]: if wanted, i can change the formatting back but lmk

File: .eslintrc.json [hunk 2]
```diff
@@ -81,9 +84,15 @@
                 ]
             }
         ],
+        "space-infix-ops": [
```
Comment (allen-tran) [inline .eslintrc.json:87]: this is the new rule

**Review by evanugarte — APPROVED** [review]
a wholesome change

**Comment by evanugarte** [comment]
this guy even included a link to js operators wow

**Review by imdominicreed — APPROVED** [review]
LGTM very clean code now 😳

**Comment by allen-tran** [comment]
> this guy even included a link to js operators wow

😆

**Pull request merged**